doc/instruction_decode_execute.rst

```diff
@@ -165,3 +162,13 @@ Source File: :file:`rtl/ibex_load_store_unit.sv`
 
 The Load-Store Unit (LSU) interfaces with main memory to perform load and store operations.
 See :ref:`load-store-unit` for more details.
+
+.. rubric:: Footnotes
+
+.. [#B_draft] Ibex fully implements draft version 0.92 of the RISC-V Bit Manipulation Extension.
+   This extension may change before being accepted as a standard by the RISC-V Foundation.
+   Ultimately, Ibex will be updated accordingly to implement the final standard extension.
```

Comment:
I think perhaps the statement is:
"Ibex will be updated to match future development of the specification, which prior to ratification may involve backwards incompatible changes to the currently implemented version. Additionally, neither GCC or Clang have committed to maintaining support upstream for unratified versions of the specification."

Reply:
Thanks @asb, I switched to your suggestion.